diff -ur ruby-mongrel-1.1.5.orig/bin/mongrel_rails ruby-mongrel-1.1.5/bin/mongrel_rails
--- ruby-mongrel-1.1.5.orig/bin/mongrel_rails	1970-01-01 01:00:00.000000000 +0100
+++ ruby-mongrel-1.1.5/bin/mongrel_rails	2009-08-08 20:13:26.256567691 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/ruby
 # Copyright (c) 2005 Zed A. Shaw
 # You can redistribute it and/or modify it under the same terms as Ruby.
 #
@@ -7,7 +8,6 @@
 require 'yaml'
 require 'etc'
 
-$LOAD_PATH.unshift "#{File.dirname(__FILE__)}/../lib"
 require 'mongrel'
 require 'mongrel/rails'
 
diff -ur ruby-mongrel-1.1.5.orig/examples/camping/blog.rb ruby-mongrel-1.1.5/examples/camping/blog.rb
--- ruby-mongrel-1.1.5.orig/examples/camping/blog.rb	1970-01-01 01:00:00.000000000 +0100
+++ ruby-mongrel-1.1.5/examples/camping/blog.rb	2009-08-08 20:13:26.246567638 +0200
@@ -1,8 +1,6 @@
 #!/usr/bin/env ruby
 
 $:.unshift File.dirname(__FILE__) + "/../../lib"
-require 'rubygems'
-require_gem 'camping', '>=1.4'
 require 'camping/session'
   
 Camping.goes :Blog
diff -ur ruby-mongrel-1.1.5.orig/examples/camping/tepee.rb ruby-mongrel-1.1.5/examples/camping/tepee.rb
--- ruby-mongrel-1.1.5.orig/examples/camping/tepee.rb	1970-01-01 01:00:00.000000000 +0100
+++ ruby-mongrel-1.1.5/examples/camping/tepee.rb	2009-08-08 20:13:26.246567638 +0200
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby
 $:.unshift File.dirname(__FILE__) + "/../../lib"
-%w(rubygems redcloth camping acts_as_versioned).each { |lib| require lib }
+%w(redcloth camping acts_as_versioned).each { |lib| require lib }
 
 Camping.goes :Tepee
 
diff -ur ruby-mongrel-1.1.5.orig/examples/mongrel_simple_service.rb ruby-mongrel-1.1.5/examples/mongrel_simple_service.rb
--- ruby-mongrel-1.1.5.orig/examples/mongrel_simple_service.rb	1970-01-01 01:00:00.000000000 +0100
+++ ruby-mongrel-1.1.5/examples/mongrel_simple_service.rb	2009-08-08 20:16:44.469881354 +0200
@@ -1,5 +1,4 @@
 # This script emualtes script/server behavior but running webrick http server 
-require 'rubygems'
 
 require 'mongrel'
 require 'yaml'
diff -ur ruby-mongrel-1.1.5.orig/lib/mongrel/command.rb ruby-mongrel-1.1.5/lib/mongrel/command.rb
--- ruby-mongrel-1.1.5.orig/lib/mongrel/command.rb	1970-01-01 01:00:00.000000000 +0100
+++ ruby-mongrel-1.1.5/lib/mongrel/command.rb	2009-08-08 20:13:26.253234425 +0200
@@ -148,6 +148,7 @@
 
       # Builds a list of possible commands from the Command derivates list
       def commands
+				return [] # Neutered
         pmgr = GemPlugin::Manager.instance
         list = pmgr.plugins["/commands"].keys
         return list.sort
@@ -190,7 +191,8 @@
             cmd_name = "mongrel::" + cmd_name
           end
 
-          command = GemPlugin::Manager.instance.create("/commands/#{cmd_name}", :argv => args)
+					# Neutered
+          # command = GemPlugin::Manager.instance.create("/commands/#{cmd_name}", :argv => args)
         rescue OptionParser::InvalidOption
           STDERR.puts "#$! for command '#{cmd_name}'"
           STDERR.puts "Try #{cmd_name} -h to get help."
diff -ur ruby-mongrel-1.1.5.orig/lib/mongrel/gems.rb ruby-mongrel-1.1.5/lib/mongrel/gems.rb
--- ruby-mongrel-1.1.5.orig/lib/mongrel/gems.rb	1970-01-01 01:00:00.000000000 +0100
+++ ruby-mongrel-1.1.5/lib/mongrel/gems.rb	2009-08-08 20:16:10.039884828 +0200
@@ -5,18 +5,10 @@
       def require(library, version = nil)
         begin
           Kernel.require library
-        rescue LoadError, RuntimeError => e
-          begin 
-            # ActiveSupport breaks 'require' by making it always return a true value
-            Kernel.require 'rubygems'
-            version ? gem(library, version) : gem(library)
-            retry
-          rescue Gem::LoadError, LoadError, RuntimeError
+                              rescue LoadError, RuntimeError
             # puts "** #{library.inspect} could not be loaded" unless library == "mongrel_experimental"
-          end
         end  
       end
-      
     end    
   end
 end
\ No newline at end of file
