--- ruby-mongrel-1.1.1-o/examples/camping/blog.rb	2007-12-12 15:22:49.000000000 -0700
+++ ruby-mongrel-1.1.1/examples/camping/blog.rb	2007-12-12 15:50:47.000000000 -0700
@@ -1,8 +1,6 @@
 #!/usr/bin/env ruby
 
 $:.unshift File.dirname(__FILE__) + "/../../lib"
-require 'rubygems'
-require_gem 'camping', '>=1.4'
 require 'camping/session'
   
 Camping.goes :Blog
--- ruby-mongrel-1.1.1-o/examples/camping/tepee.rb	2007-12-12 15:22:49.000000000 -0700
+++ ruby-mongrel-1.1.1/examples/camping/tepee.rb	2007-12-12 15:50:47.000000000 -0700
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby
 $:.unshift File.dirname(__FILE__) + "/../../lib"
-%w(rubygems redcloth camping acts_as_versioned).each { |lib| require lib }
+%w(redcloth camping acts_as_versioned).each { |lib| require lib }
 
 Camping.goes :Tepee
 
--- ruby-mongrel-1.1.1-o/examples/mongrel_simple_service.rb	2007-12-12 15:22:49.000000000 -0700
+++ ruby-mongrel-1.1.1/examples/mongrel_simple_service.rb	2007-12-12 15:50:47.000000000 -0700
@@ -1,5 +1,4 @@
 # This script emualtes script/server behavior but running webrick http server 
-require 'rubygems'
 
 require 'mongrel'
 require 'yaml'
--- ruby-mongrel-1.1.1-o/lib/mongrel/command.rb	2007-12-12 15:22:50.000000000 -0700
+++ ruby-mongrel-1.1.1/lib/mongrel/command.rb	2007-12-12 15:50:47.000000000 -0700
@@ -148,6 +148,7 @@
 
       # Builds a list of possible commands from the Command derivates list
       def commands
+				return [] # Neutered
         pmgr = GemPlugin::Manager.instance
         list = pmgr.plugins["/commands"].keys
         return list.sort
@@ -190,7 +191,8 @@
             cmd_name = "mongrel::" + cmd_name
           end
 
-          command = GemPlugin::Manager.instance.create("/commands/#{cmd_name}", :argv => args)
+					# Neutered
+          # command = GemPlugin::Manager.instance.create("/commands/#{cmd_name}", :argv => args)
         rescue OptionParser::InvalidOption
           STDERR.puts "#$! for command '#{cmd_name}'"
           STDERR.puts "Try #{cmd_name} -h to get help."
--- ruby-mongrel-1.1.1-o/lib/mongrel/gems.rb	2007-12-12 15:22:50.000000000 -0700
+++ ruby-mongrel-1.1.1/lib/mongrel/gems.rb	2007-12-12 15:51:08.000000000 -0700
@@ -5,18 +5,11 @@
       def require(library, version = nil)
         begin
           Kernel.require library
-        rescue LoadError, RuntimeError => e
-          begin 
-            # ActiveSupport breaks 'require' by making it always return a true value
-            require 'rubygems'
-            version ? gem(library, version) : gem(library)
-            retry
-          rescue Gem::LoadError, LoadError, RuntimeError
+				rescue LoadError, RuntimeError
             # puts "** #{library.inspect} could not be loaded" unless library == "mongrel_experimental"
-          end
         end  
       end
       
     end    
   end
-end
\ No newline at end of file
+end
